noxfiles/docker_nox.py

```diff
@@ -95,6 +96,7 @@ def build(session: nox.Session, image: str, machine_type: str = "") -> None:
         session.log(f"  - {privacy_center_image_tag}")
         session.run(
             "docker",
+            "buildx",
```

Comment:
This causes _every_ build that a user does to build both images. This takes a _very_ long time (even on my beefed up machine) and is probably not what we want for a developer experience. I'm going to refactor `push` a bit so that it also handles the builds, I think this is a fair tradeoff so we can keep `dev` light and let `push` handle the heavier building in addition to pushing